Release/1.0.0

cs-abhinav-goel opened the pull request:
Initial Draft

cs-abhinav-goel committed:
Update year in License

cs-abhinav-goel committed:
Initial Draft

cs-pranali-chavan commented on widget/view.controller.js:
```diff
@@ -0,0 +1,97 @@
+    getPicklistValues();
+    $scope.activeItemImage = currentTheme === 'light' ? 'widgets/installed/infoGraphics-1.0.0/images/chevron_inactive_arrow.png' : 'widgets/installed/infoGraphics-1.0.0/images/chevron_active_arrow.png';
+    $scope.inactiveItemImage = currentTheme === 'light' ? 'widgets/installed/infoGraphics-1.0.0/images/chevron_active_arrow.png' : 'widgets/installed/infoGraphics-1.0.0/images/chevron_inactive_arrow.png';
+  }
```
import & use widgetBasePath 
widgetBasePath +'images/chevron_inactive_arrow.png'

cs-abhinav-goel replied:
Done

cs-pranali-chavan reviewed (changes requested):
use widgetBasePath for version updates

cs-suyog-jinturkar commented on widget/info.json:
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "infoGraphics",
+    "title": "Info Graphics",
```
snapshots will be added here as well which will show edit and view panel screenshots

cs-abhinav-goel committed:
Update the code to use widgetBasePath

cs-abhinav-goel committed:
Changed Name of the connectors & added tooltip

Below changes done against this commit:
1. Changed name of the widget
2. Added tooltip for field in edit form
3. Added images for the white theme
